distributionviewer/core/static/js/app/router.js

```diff
@@ -6,13 +6,13 @@ import MainLayout from './components/layouts/main-layout';
 
 // Pages
 import Home from './components/home';
-import ExampleChartDetailContainer from './components/containers/example-chart-detail-container';
+import ChartDetailContainer from './components/containers/chart-detail-container';
 
 export default (
   <Router history={browserHistory}>
     <Route component={MainLayout}>
       <Route path="/" component={Home} />
-      <Route path="/chart/:chartId" component={ExampleChartDetailContainer} />
+      <Route path="/chart/:chartName" component={ChartDetailContainer} />
```

Comment:
This will give us paths like */chart/architecture/*. Not sure if *chart* is the best name here. *metric*? *graph*? Open to suggestions.